### PR TITLE
Update gatsby-theme-aio in package.json

### PR DESCRIPTION
**Pull request by hsalhotr**
After this fix https://github.com/AdobeDocs/static-website-deploy/pull/1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/icaraps"
   },
   "dependencies": {
-    "@adobe/gatsby-theme-aio": "4.9.12",
+    "@adobe/gatsby-theme-aio": "^4.13.0",
```
Comment (adityaganotra) [inline package.json:15]: is this change necessary ?

Comment (hsalhotr) [reply package.json:15]: Yeah reverted and checked. Deployment is successful without this change as well.
